op-chain-ops/foundry/allocs_test.go

```diff
@@ -58,15 +58,15 @@ func TestForgeAllocs_FromState(t *testing.T) {
 
 	// Commit and make a new state, we cannot reuse the state after Commit
 	// (see doc-comment in Commit, absolute footgun)
-	root, err := st.Commit(0, false)
+	root, err := st.Commit(0, false, false) // TODO: false ok here? In general, should be IsCancun()
```

Comment:
TBD

Reply:
It's `false` in the geth `genesis.go`, so let's go with that. Pruning empty accounts from genesis allocs makes sense to me.